File: Week 14/CS/Circle.cs
using System;

namespace ConsoleApp
{
	public class Circle : Shape {
		private double radius;

		public Circle(double r) {
			radius = r;
		}
		public void SetRadius(double r) {
			radius = r;
		}

		public double GetArea() {
			return radius * radius * 3.1415;
		}

		public override String SayHello() {
			return "Hello (Circle)";
		}
	}
}

File: Week 14/Java/Circle.java
package lastSession;

public class Circle extends Shape {
	private double radius;
	
	public Circle(double r) {
		radius = r;
	}
	public void setRadius(double r) {
		radius = r;
	}
	public double getArea() {
		return radius * radius * 3.1415;
	}
	@Override
	public String sayHello() {
		return "Hello (Circle)";
	}
}
